internal/cmd/githubtemplatefuncs.go

```diff
@@ -25,23 +25,30 @@ type gitHubLatestReleaseState struct {
 	Release     *github.RepositoryRelease `json:"release"     yaml:"release"`
 }
 
-type gitHubLatestTagState struct {
-	RequestedAt time.Time             `json:"requestedAt" yaml:"requestedAt"`
-	Tag         *github.RepositoryTag `json:"tag"         yaml:"tag"`
+type gitHubReleasesState struct {
+	RequestedAt time.Time                   `json:"requestedAt" yaml:"requestedAt"`
+	Releases    []*github.RepositoryRelease `json:"release"     yaml:"release"`
```

Comment:
Nit: the tags should be `json:"releases" yaml:"releases"` (plural, not singular).